### PR TITLE
Add types for bnjs

### PR DESCRIPTION
**Pull request by dummytester123**
Ok, been working on the new yarn2-based monorepository over at https://github.com/solana-labs/oyster. Had an issue where u64 type imported from spl-token lacked any typing.

It has to do with yarn 2 being way smarter with dependencies than npm ever was. The issue is that in the spl-token cli,, we never added "@types/bn.js":"^5.1.0" to the devDependencies. In NPM projects, this is fine as long as you add this to the top level package.json (which we do in oyster-lending) but in yarn2, it's smart enough to not pull that from the outer scope and instead just has no type for the inner scope. We need to release a version with this package.json (name it 0.0.14) and then the monorepo will work.

@joncinque @bartosz-lipinski

### CODE REVIEW

File: token/js/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana/spl-token",
-  "version": "0.0.13",
+  "version": "0.0.14",
```
Comment (bartosz-lipinski) [inline token/js/package.json:3]: I think we could move this package to 0.1.0, since there is a lot of projects depending on it

**Review by bartosz-lipinski — APPROVED** [review]
Awesome great catch.

**Comment by jstarry** [comment]
LGTM, add this flag to fix CI
```
tsc --esModuleInterop module.d.ts
```

**Comment by dummytester123** [comment]
@jstarry where do I add this? To some script in package.json? I'm not familiar with the CI setup here yet.

**Comment by jstarry** [comment]
In .Travis/script.sh

**Comment by jstarry** [comment]
@dummytester123 thanks! Sorry I mixed up repo this was, glad you found the script. Could you also fix the conflicts please?

**Comment by dummytester123** [comment]
Done!

File: token/js/package.json [hunk 2]
```diff
@@ -42,8 +42,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.10.5",
-    "@solana/web3.js": "^0.91.0",
-    "bn.js": "^5.0.0",
+    "@solana/web3.js": "^0.90.0",
```
Comment (jstarry) [inline token/js/package.json:45]: Should be 0.91

File: token/js/package.json (hunk 1 above)
Comment (jstarry) [reply token/js/package.json:3]: @dummytester123 can you bump to 0.1.0 like bartosz suggested?

File: token/js/package.json (hunk 2 above)
Comment (dummytester123) [reply token/js/package.json]: done

**Comment by jstarry** [comment]
@dummytester123 I think you missed this comment: https://github.com/solana-labs/solana-program-library/pull/1194#discussion_r572459417

File: token/js/package.json (hunk 1 above)
Comment (dummytester123) [reply token/js/package.json:3]: oh, totally missed that.

**Pull request merged**